### PR TITLE
[easy] `ethcore-bloom-journal` was renamed to `accounts-bloom`

### LINKED ISSUE #11585: Rename `ethcore-bloom-journal`
Rename `ethcore-bloom-journal` to `accounts-bloom`.

### PR DESCRIPTION
**Pull request by shergin**
It fixes #11585.

Test Plan: It compiles. `¯\_(ツ)_/¯`

### CODE REVIEW

**Comment by shergin** [comment]
cc @dvdplm

File: CHANGELOG.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ The full list of included changes from `v2.5-stable` to `v2.7-stable` (the
 * removed EthEngine alias (#10805)
 * wait a bit longer in should_check_status_of_request_when_its_resolved (#10808)
 * Do not drop the peer with None difficulty (#10772)
-* ethcore-bloom-journal updated to 2018 (#10804)
+* accounts-bloom updated to 2018 (#10804)
```
Comment (niklasad1) [inline CHANGELOG.md:265]: please revert this, we don't want to modify the `CHANGELOG` of an old release.

File: docs/CHANGELOG-2.6.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ The full list of included changes:
 * revert temp changes to .gitlab-ci.yml ([#10807](https://github.com/openethereum/openethereum/pull/10807))
 * removed redundant fmt::Display implementations ([#10806](https://github.com/openethereum/openethereum/pull/10806))
 * removed EthEngine alias ([#10805](https://github.com/openethereum/openethereum/pull/10805))
-* ethcore-bloom-journal updated to 2018 ([#10804](https://github.com/openethereum/openethereum/pull/10804))
+* accounts-bloom updated to 2018 ([#10804](https://github.com/openethereum/openethereum/pull/10804))
```
Comment (niklasad1) [inline docs/CHANGELOG-2.6.md:187]: please revert this, we don't want to modify the CHANGELOG of an old release.

**Comment by shergin** [comment]
Thank you!
Done!

**Pull request merged**